decryptor/postgresql/pg_escape_decryptor.go

```diff
@@ -99,7 +106,7 @@ func (decryptor *PgEscapeDecryptor) readOctalData(data, octData []byte, reader i
 			return dataIndex, octDataIndex, err
 		}
 		if n != 1 {
-			log.Debugln("readOctalData read 0 bytes")
+			decryptor.logger.Debugln("readOctalData read 0 bytes")
```

Comment:
As I understand, now we write logs from capital letter

Reply:
yes but I leaved `readOctalData` as is because it's name of method